_posts/newsletters/2021-05-22-newsletter.md

```diff
@@ -0,0 +1,115 @@
+* [Get Involved](#involved)
+
+<a name="updates"></a>
+# **Updates on Working Groups**
+
+**Education & Training**: This working group is for those that are interested in regular discussions about education & training for RSEs with a goal of developing concrete deliverables. The group meets on the first Monday of the month at 12:00 Eastern. Thus far, we have begun describing the types of people who work as RSEs and what their training needs might be and we are working on defining the list of skills often needed by RSEs. If you would like to join the conversation, please join the [#education-training](https://usrse.slack.com/messages/education-training) channel on the US-RSE Slack. You can post a message there if you would like to be invited to the next meeting.
+
+**Diversity, Equity and Inclusion (DEI)**: We are organizing a [book club / discussion group](https://us-rse.org/events/2021/2021-01-dei-book-club). The first book we're going to discuss is "Better Allies: Everyday Actions to Create Inclusive, Engaging Workplaces'' by Karen Catlin. We invite anyone interested to please [sign up and join us](https://www.eventbrite.com/e/better-allies-book-discussion-part-i-tickets-151687323869) on Tuesday, May 25 from 12:00 PM -12:50 PM ET for a discussion of "Part One: Watching and Observing as an Ally'' (pages 1-55). We're also planning the next event in our [Speaker Series](https://us-rse.org/events/2021/2021-03-dei-speaker-series) and finalizing an update of the US-RSE mission statement to better address DEI issues. Join the effort at the [#dei-discussion](https://usrse.slack.com/archives/C01C8CJQ7AP) channel on Slack.
+
+**Website**: The website working group is working on a new design for the us-rse.org website. The group is meeting bi-weekly and a beta version of a new website is in development. The new site promises to have better functionality and usability while still retaining the international RSE community feel. If you’d like to contribute please join the[ #website](https://usrse.slack.com/messages/website) channel in the US-RSE Slack to join in the conversation.
+
+<a name="workshop"></a>
+# **2021 US-RSE Virtual Workshop**
+
+Following the success of the 2020 US-RSE Virtual Workshop, the event this year is also virtual due to the ongoing pandemic. The [2021 US-RSE Virtual Workshop](https://us-rse.org/virtual-workshop-2021/) will be held on two non-consecutive half days: May 24 and 27, 2021, 1-4pm ET/12-3pm CT/10am-1pm PT.
+
+The theme of the workshop is "A Path forward for Research Software Engineers." The workshop will feature talks, community events, and working group sessions. It will be held via Zoom and some parts, such as the talks, will be recorded and available after the workshop via the [US-RSE YouTube channel](https://www.youtube.com/channel/UC7IQsWv809OQYJ-sJKuQZrw). The workshop is free but [registration is required, here](https://docs.google.com/forms/d/e/1FAIpQLSetiGdTiHSTfil4-LL_R3eZEyNIIfGtSJqjTsViBPcldsJtJw/viewform).
```

Comment:
```suggestion
The theme of the workshop is "A Path Forward for Research Software Engineers." The workshop will feature talks, community events, and working group sessions. It will be held via Zoom and some parts, such as the talks, will be recorded and available after the workshop via the [US-RSE YouTube channel](https://www.youtube.com/channel/UC7IQsWv809OQYJ-sJKuQZrw). The workshop is free but [registration is required, here](https://docs.google.com/forms/d/e/1FAIpQLSetiGdTiHSTfil4-LL_R3eZEyNIIfGtSJqjTsViBPcldsJtJw/viewform).
```

Reply:
I think this is a good change to accept